app/models/public_body.rb

```diff
@@ -75,6 +75,9 @@ class ImportCSVDryRun < StandardError ; end
   after_update :reindex_requested_from
 
 
+  # Force eager loading of translations everywhere
+  default_scope { includes(:translations) }
```

Comment:
Not sure, went off the Rails docs which suggest the syntax I used: http://api.rubyonrails.org/v4.2/classes/ActiveRecord/Scoping/Default/ClassMethods.htmll
http://api.rubyonrails.org/v5.1/classes/ActiveRecord/Scoping/Default/ClassMethods.html

(Followup comment - on that SO thread reveals the user to have been running MongoDB via the Mongoid gem rather than relying on ActiveRecord)